src/components/ServiceRequestForm.jsx

```diff
@@ -0,0 +1,127 @@
+		const { value } = changeEvent.currentTarget;
+		const subCategories = getSubCategories(categories, parseInt(value));
+		setFieldValue(changeEvent.currentTarget.name, changeEvent.currentTarget.value)
+		setFieldValue('subRequestType', '')
+		setSubCategories(subCategories);
+	};
+
+
+	return (
+
+		<React.Fragment>
+			<FormContainer title="How Can We Help?">
+				<Formik
+					initialValues={{
+						requestType: '',
+						subRequestType: '',
+					}}
+					validationSchema={Yup.object().shape({
+						requestType: Yup.string().required('Request Category is required'),
+						subRequestType: Yup.string().required('Sub Category is required'),
+					})}
+
+					onSubmit={(values, { setSubmitting }) => {
+
+						alert(JSON.stringify(values, null, 2));
+						setSubmitting(false);
+
+					}}
+				>
+					{
+						(props) => {
+							const { values, isSubmitting, errors, touched } = props;
+
+							return (
+
+								<Form >
+									<label htmlFor="requestType"
+										className={
+											errors.requestType && touched.requestType ? "input-feedback" : "text-label"}
+
+									>Request Category</label>
+									<br />
+									<Field
+										component="select"
+										name="requestType"
+										onChange={e => { handleServiceRequestChange(e, props.setFieldValue) }}
+										className={errors.requestType && touched.requestType ? "text-select error" : "text-select"}
+									>
+										<option key='default' value=''>--Please select a category--</option>
+										{Categories.map(category => (
+											<option key={category.id} value={category.id}>{category.name}</option>
+										))}
+									</Field>
+
+									<div className="input-feedback">
+										<ErrorMsg
+											errormessage={errors.requestType}
+											touched={touched.requestType} />
+									</div>
+
+
+									{
+										values['requestType'] !== '' ?
+											<div>
+												<label name="subRequestType" htmlFor="subRequestType"
+													className={
+														errors.subRequestType && touched.subRequestType ? "input-feedback" : "text-label"}
+												>
+													Request Sub-Category
+												</label>
+												<br />
+												<Field component="select"
+													name="subRequestType"
+												>
+													<option key='default' value=''>--Please Select a sub-category--</option>;
+													{subCategories.map(category => (
+														<option key={category.id} value={category.id}>{category.name}</option>
+													))}
+												</Field>
+												<div className="input-feedback">
+													<ErrorMsg
+														errormessage={errors.subRequestType}
+														touched={touched.subRequestType} />
+												</div>
+											</div>
+											: null
+									}
+									<br />
+									<button type="submit" disabled={isSubmitting}>
+										Submit
+									</button>
+									<br />
+									<h6>Why do I need this </h6>
```

Comment:
you can remove the <br> <h6> and the commented code, this will go in a different component.

Reply:
css has been removed